pkg/job/local_job.go

```diff
@@ -68,6 +69,7 @@ func (j *localJob) GenerateInstallConfig(ctx context.Context, cluster common.Clu
 		"aws_access_key_id="+j.Config.AwsAccessKeyID,
 		"aws_secret_access_key="+j.Config.AwsSecretAccessKey,
 		"WORK_DIR=/data",
+		"SKIP_CERT_VERIFICATION="+strconv.FormatBool(j.Config.SkipCertVerification),
```

Comment:
I think we should also add the `CA_CERT_PATH` variable, with an empty values.
It will make it easier to change in the future.
Also the value for SKIP_CERT_VERIFICATION here should always be "false" 
We said that for now the job also runs without the certificate, even when running on Openshift

Reply:
`SKIP_CERT_VERIFICATION=false` will fail if the CA is unrecognized or the SAN is incorrect. So did you mean `SKIP_CERT_VERIFICATION` should be always `true`? I think best practices even for internal traffic to be TLS. I agree regarding CA_CERT_PATH, but it was de-scoped for now.

Reply:
How will this work in OC installation?
We'll use verification, but the job will not have the certificated mounted.